vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/MultiSelectComboBox.java

```diff
@@ -395,25 +398,79 @@ public void updateSelection(Set<TItem> addedItems,
     }
 
     /**
-     * Gets whether displaying of all selected item labels as chips is enabled.
+     * Defines possible behavior of the component when not all selected items
+     * can be displayed as chips within the current field width.
+     */
+    public enum AutoExpandMode {
+        /**
+         * Field expands vertically and chips wrap.
+         */
+        VERTICAL,
+
+        /**
+         * Field expands horizontally until max-width is reached, then collapses
+         * to overflow chip.
+         */
+        HORIZONTAL,
+
+        /**
+         * Field expands horizontally until max-width is reached, then expands
+         * vertically and chips wrap.
+         */
+        BOTH,
+
+        /**
+         * Field does not expand and collapses to overflow chip.
+         */
+        NONE,
+    }
+
+    /**
+     * Gets the behavior of the component when not all selected items can be
+     * displayed as chips within the current field width.
      *
      * @since 24.3
-     * @return {@code true} if enabled, {@code false} otherwise
+     * @return {AutoExpandMode}
```

Comment:
The `@return` documentation doesn't look right:
<img width="897" alt="Screenshot 2023-11-14 at 9 37 45" src="https://github.com/vaadin/flow-components/assets/1222264/26807a87-5765-4eb0-8554-03f7b1c1ca4d">

Could be something like `* @return The current {@link AutoExpandMode}`

Reply:
Thanks, changed.

Reply:
It's still missing the `@link` from inside the brackets

Currently:
<img width="260" alt="Screenshot 2023-11-14 at 10 13 13" src="https://github.com/vaadin/flow-components/assets/1222264/8c879667-ef9d-4e70-90f5-48902fee5fa4">

Should be:
<img width="248" alt="Screenshot 2023-11-14 at 10 16 52" src="https://github.com/vaadin/flow-components/assets/1222264/8bd1b427-75b7-45a2-89f1-c7a8ea8a5d2d">

Reply:
Fixed.